mmeval/metrics/one_minus_norm_edit_distance.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import re
+from typing import TYPE_CHECKING, Dict, List, Sequence
+
+from mmeval.core import BaseMetric
+from mmeval.utils import try_import
+
+if TYPE_CHECKING:
+    from rapidfuzz.distance import Levenshtein
+else:
+    distance = try_import('rapidfuzz.distance')
+    if distance is not None:
+        Levenshtein = distance.Levenshtein
+
+
+class OneMinusNormEditDistance(BaseMetric):
+    r"""One minus NED metric for text recognition task.
+
+    Args:
+        letter_case (str): There are three options to alter the letter cases
+
+            - unchanged: Do not change prediction texts and labels.
+            - upper: Convert prediction texts and labels into uppercase
+              characters.
+            - lower: Convert prediction texts and labels into lowercase
+              characters.
+
+            Usually, it only works for English characters. Defaults to
+            'unchanged'.
+        invalid_symbol (str): A regular expression to filter out invalid or
+            not cared characters. Defaults to '[^A-Za-z0-9\u4e00-\u9fa5]'.
+        **kwargs: Keyword parameters passed to :class:`BaseMetric`.
+
+    Examples:
+        >>> from mmeval import OneMinusNormEditDistance
+        >>> metric = OneMinusNormEditDistance()
+        >>> metric(['helL', 'HEL'], ['hello', 'HELLO'])
+        {'1-N.E.D': 0.6}
+        >>> metric = OneMinusNormEditDistance(letter_case='upper')
+        >>> metric(['helL', 'HEL'], ['hello', 'HELLO'])
+        {'1-N.E.D': 0.7}
+    """
+
+    def __init__(self,
+                 letter_case: str = 'unchanged',
+                 invalid_symbol: str = '[^A-Za-z0-9\u4e00-\u9fa5]',
+                 **kwargs):
+        super().__init__(**kwargs)
+
```

Comment:
`rapidfuzz` is not added in `runtime.txt` so the metric will throw an error when calling `add` method. The solution is to check it in `__init__`.